src/pages/workspace/expensifyCard/WorkspaceEditCardLimitTypePage.tsx

```diff
@@ -41,28 +46,47 @@ function WorkspaceEditCardLimitTypePage({route}: WorkspaceEditCardLimitTypePageP
     const {translate} = useLocalize();
     const styles = useThemeStyles();
 
-    const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${policyID}_${CONST.EXPENSIFY_CARD.BANK}`);
+    const [cards] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${policyID}_${CONST.EXPENSIFY_CARD.BANK}`);
     const policy = usePolicy(policyID);
 
-    const card = cardsList?.[cardID] ?? mockedCard;
+    const card = cards?.[cardID] ?? mockedCard;
     const areApprovalsConfigured = !isEmptyObject(policy?.approver) && policy?.approvalMode !== CONST.POLICY.APPROVAL_MODE.OPTIONAL;
     const defaultLimitType = areApprovalsConfigured ? CONST.EXPENSIFY_CARD.LIMIT_TYPES.SMART : CONST.EXPENSIFY_CARD.LIMIT_TYPES.MONTHLY;
     const initialLimitType = card.nameValuePairs?.limitType ?? defaultLimitType;
     const promptTranslationKey =
         initialLimitType === CONST.EXPENSIFY_CARD.LIMIT_TYPES.MONTHLY
-            ? 'workspace.expensifyCard.changeCardMonthlyLimitTypeWarning'
-            : 'workspace.expensifyCard.changeCardSmartLimitTypeWarning';
+            ? 'workspace.expensifyCard.changeCardSmartLimitTypeWarning'
+            : 'workspace.expensifyCard.changeCardMonthlyLimitTypeWarning';
```

Comment:
Oh yes, it was an issue with translation namings (it was reverted) and forgot to change

Reply:
If we have initialLimitType as CONST.EXPENSIFY_CARD.LIMIT_TYPES.MONTHLY then we need to have SmartLimit warning because we are changing Monthly -> SmartLimit and warning will be for smartLimit and vise versa